aio/content/examples/testing/src/app/demo/demo.testbed.spec.ts

```diff
@@ -273,9 +273,10 @@ describe('demo (with TestBed):', () => {
         expect(comp.name).toBe(expectedOrigName,
           `comp.name should still be ${expectedOrigName} after value change, before binding happens`);
 
-        // dispatch a DOM event so that Angular learns of input value change.
+        // Dispatch a DOM event so that Angular learns of input value change.
         // then wait while ngModel pushes input.box value to comp.name
-        input.dispatchEvent(newEvent('input'));
+        // In older browsers, such as IE, you might need a CustomEvent instead, see https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill
```

Comment:
This long line will not look good in the code-snippets: 

![new-event-wrapped](https://user-images.githubusercontent.com/8604205/83509322-eed1c100-a4d3-11ea-9d77-c9622cdee328.png)

Please change it to something like (here and in the other files):

```suggestion
        // In older browsers, such as IE, you might need a CustomEvent instead. See
        // https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill
```

Reply:
done